code/modules/paperwork/faxmachine.dm

```diff
@@ -330,13 +330,56 @@ GLOBAL_LIST_EMPTY(fax_blacklist)
 	switch(destination)
 		if("Central Command")
 			message_admins(sender, "CENTCOM FAX", destination, copyitem, "#006100")
+			notify_ghosts("[sender] has sent a fax to [destination].", 'sound/goonstation/machines/printer_dotmatrix.ogg', "<a href='?_src_=holder;GhostFaxView=\ref[copyitem]'>view message</a>", "Fax sent", src)
 		if("Syndicate")
 			message_admins(sender, "SYNDICATE FAX", destination, copyitem, "#DC143C")
+			notify_ghosts("[sender] has sent a fax to the [destination].", 'sound/goonstation/machines/printer_dotmatrix.ogg', "<a href='?_src_=holder;GhostFaxView=\ref[copyitem]'>view message</a>", "Fax sent", src)
 	for(var/obj/machinery/photocopier/faxmachine/F in GLOB.allfaxes)
 		if(F.department == destination)
 			F.receivefax(copyitem)
 	visible_message("[src] beeps, \"Message transmitted successfully.\"")
 
+/obj/machinery/photocopier/faxmachine/Topic(href, href_list)
+	if(href_list["GhostFaxView"])
+		var/mob/dead/observer/ghost = usr
+		if(istype(ghost))
```

Comment:
You've got this `ghost` variable here, but you're not actually using it for anything except `istype(ghost)`.
Should probably either just do `isobserver(usr)` or replace the other `usr` arguments below with `ghost`.

Reply:
Done!